src/common/Navigation/index.js

```diff
@@ -1,5 +1,6 @@
 import { Search } from "./Search";
 import { useLocation } from "react-router-dom";
+import { Link } from "react-router-dom/cjs/react-router-dom.min";
```

Comment:
could someone explain me, why VSC returns such a strange path as default import ? :D  In my files I'm changing it manually, to just "react-router-dom", since it works both ways... I think it's more elegant :D

Reply:
In my experience, for some reason it just always imports most of the tools we use from react-router in this way, obviously works both ways, but I didn't bother myself yet to find the exact reason why it happens since it's really irrevelant comparing to the other issues we're dealing with.